client/src/client.cpp

```diff
@@ -0,0 +1,28 @@
+//
+// Created by andykong on 22-6-22.
+//
+#include <ros/ros.h>
+#include "client/name.h"
+#include "client/topic_name.h"
+
+int main(int argc, char **argv) {
+    ros::init(argc, argv, "client");
+    ros::NodeHandle n;
+    ros::service::waitForService("server");
+    ros::ServiceClient client_name = n.serviceClient<client::topic_name>("server");
+    client::topic_name srv;
+    srv.request.name = "client";
+    client_name.call(srv);
+    ROS_INFO("Result: %s", srv.response.result.c_str());
+    ros::Publisher name_info_pub = n.advertise<client::name>(srv.response.result.c_str(), 10);
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]